2022/typescript/day03.ts

```
@@ -1,39 +1,39 @@
-};
-console.log("Sum of priorities:", prioritySum());
+    .reduce((acc, cur) => acc + cur, 0)
+}
+console.log('Sum of priorities:', prioritySum())
 
 // Part 2
 export const badgesSum = () => {
-  let badgesSum = 0;
-  const add = (code: number) => (badgesSum += code >= 97 ? code - 96 : code - 38);
+  let badgesSum = 0
+  const add = (code: number) => (badgesSum += code >= 97 ? code - 96 : code - 38)
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)